.github/workflows/gnitpick.yml

```diff
@@ -0,0 +1,27 @@
+name: 'Nitpick git commits'
+
+on: [push, pull_request]
+
+env:
+  PYTHON_VERSION: 3.6
+
+jobs:
+  nitpick:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup repository
+        uses: actions/checkout@v2
+
+      - name: Set up python ${{ env.PYTHON_VERSION }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+
+      - name: Install dependencies
+        run: pip install flake8
+
+      - name: Run gnitpick
+        run: python gnitpick.py --verbose
```

Comment:
You should use `python3` so it's explicit that we want to use that version. `python` might refer to `python2` or `python3`.

Reply:
Updated